komodoenv/__main__.py

```diff
@@ -97,9 +105,16 @@ def parse_args(args):
         "-r",
         "--release",
         type=str,
-        required=False,
+        default=os.environ.get("KOMODO_RELEASE", "bleeding"),
```

Comment:
Just wandering whether default shouldn't be stable

Reply:
That is an interesting point. I used `bleeding` because I reckon the happy paths are:
1. Users of `komodo`, who source it first and therefore `KOMODO_RELEASE` is set and this default doesn't apply.
2. Developers/testers of `komodoenv`, who install this project into a separate venv without sourcing `komodo`. Because this is an advanced strategy, I figured having `bleeding` be the default is reasonable.

Reply:
Now it makes sense :+1: